WebAssembly/Runtime/UnmanagedMemory.cs

```diff
@@ -16,8 +17,56 @@ public sealed class UnmanagedMemory : IDisposable
             => typeof(UnmanagedMemory).GetTypeInfo().DeclaredProperties.Where(prop => prop.Name == nameof(Start)).First().GetMethod);
         internal static readonly RegeneratingWeakReference<MethodInfo> GrowMethod = new RegeneratingWeakReference<MethodInfo>(()
             => typeof(UnmanagedMemory).GetTypeInfo().DeclaredMethods.Where(prop => prop.Name == nameof(Grow)).First());
-        private static readonly RegeneratingWeakReference<long[]> emptyPage = new RegeneratingWeakReference<long[]>(() => new long[Memory.PageSize / 8]);
 
+        /// <summary>
+        /// Sets a <paramref name="length"/> of bytes to zero at a <paramref name="destination"/> memory location.
+        /// </summary>
+        /// <param name="destination">The write target memory location.</param>
+        /// <param name="length">The count of bytes to be copied.</param>
+        public delegate void ZeroMemoryDelegate(IntPtr destination, uint length);
+
+        /// <summary>
+        /// Sets a length of bytes to the given value at a destination memory location.
+        /// See <see cref="ZeroMemoryDelegate"/> for details.
+        /// </summary>
+        public static readonly ZeroMemoryDelegate ZeroMemory;
```

Comment:
I'd still like to see these two members become non-public.  The main reason for this is so we have the option to change/remove them later without risking breakage of users.